z2/resources/chain-specs/zq2-protomainnet.toml

```diff
@@ -10,7 +10,7 @@ eth_chain_id = 32770
 allowed_timestamp_skew = { secs = 60, nanos = 0 }
 data_dir = "/data"
 consensus.genesis_accounts = [ ["0x6f5d14cbe74547124321f34ea14016f96372c4ae", "900_000_000_000_000_000_000_000_000" ] ]
-consensus.genesis_deposits = [ ["a52ad5b0bb67185541570a0b7372629bb8201278dd7c564621c6c04b8caf46502d4bceb2b48031f231d655120fa76627", "12D3KooWMoQviBdwqZuFrZXCLWpgftkwVZqbcf7zYYPQG9PzV4ub", "20_000_000_000_000_000_000_000_000", "0x0000000000000000000000000000000000000000", "0x6f5d14cbe74547124321f34ea14016f96372c4ae"] ]
+consensus.genesis_deposits = [ ["90540bf372801ed6dfdbbf1215a408f67b241035679e159b5f135d2c051678ed645e96c2ba3c08b9846d9f8c096a2832", "12D3KooWAc4NnhY8PbMaMqLTUpKz1KjJbkwxtuBX433fWH2FUavR", "20_000_000_000_000_000_000_000_000", "0x0000000000000000000000000000000000000000", "0x6f5d14cbe74547124321f34ea14016f96372c4ae"] ]
```

Comment:
We can't change the genesis block for proto-mainnet and proto-testnet. It already exists.